content/blog/2019-08-18-introducing-the-rust-game-development-working-group.md

```diff
@@ -1,6 +1,8 @@
 +++
 title = "Introducing the Rust Game Development Working Group"
-aliases = ["/2019/08/18/introducing-the-rust-game-development-working-group", "/2019/08/18/introducing-the-rust-game-development-working-group.html"]
```

Comment:
Please can you add these 2 aliases back to the list? They're there for compatibility with the old Jekyll URLs.

Reply:
Oops, missed that one.